src/seer_utils.erl

```diff
@@ -0,0 +1,72 @@
+-module(seer_utils).
+
+-include("seer.hrl").
+
+-export([carbon_format/1]).
+
+carbon_format(Metrics) ->
+    Timestamp = erlang:timestamp(),
+    lists:flatten([carbon_string(Metric, Timestamp) || Metric <- Metrics]).
+
+% private
+carbon_string({Type, Name, Value}, Timestamp) ->
+    MetricKey = metric_key(Name),
+    TimestampBin = unix_time_binary(Timestamp),
+    case Type of
+        counter ->
+            carbon_string(MetricKey, integer_to_binary(Value), TimestampBin);
+        gauge ->
+            carbon_string(MetricKey, integer_to_binary(Value), TimestampBin);
+        dist ->
+            [
+                carbon_string(
```

Comment:
Steamroller is merciless here. I think it might be more readable if you match this and the following list comprehension to their own variables then cons them both.